samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js

```diff
@@ -282,7 +282,9 @@
       $.getJSON(routePrefix + "getTimerProgress" + routeSuffix, ajaxQuery, function(data) {
           totalTime = data[0];
           elapsedTime = data[1];
-          if (totalTime === elapsedTime) {
+          submissionStatus = data[2];
+
+          if (submissionStatus == 0 || totalTime === elapsedTime) {
```

Comment:
Main mod is here is adding submissionStatus to the getTimerProgress XHR call so that there is no chance a user can background a tab and then come back to an already-submitted/already-expired exam and continue working on it